fcrepo-auth-webac/src/test/java/org/fcrepo/auth/webac/WebACAuthorizationTest.java

```diff
@@ -53,9 +55,10 @@ public void testLists() {
         final List<URI> modes = Arrays.asList(WEBAC_MODE_READ, WEBAC_MODE_WRITE, WEBAC_MODE_READ);
         final List<String> accessTo = Arrays.asList(ACCESS_TO1, ACCESS_TO2, ACCESS_TO3);
         final List<String> accessToClass = Arrays.asList(ACCESS_TO_CLASS1, ACCESS_TO_CLASS2);
+        final List<String> accessGroups = Arrays.asList(ACCESS_GROUP1, ACCESS_GROUP2);
```

Comment:
Although it is trivial, we should add assertions in these tests along the lines of:
```
assertEquals(2, auth.getAgentGroups().size());
assertTrue(auth.getAgentGroups().contains(ACCESS_GROUP2));
```

Reply:
That'll be easy enough to add.